cmd/libs/go2idl/set-gen/main.go

```diff
@@ -44,6 +46,8 @@ func main() {
 		generators.DefaultNameSystem(),
 		generators.Packages,
 	); err != nil {
-		glog.Fatalf("Error: %v", err)
+		glog.Errorf("Error: %v", err)
```

Comment:
I don't actually recall changing this-- maybe I was paranoid about the exit code?